### PR TITLE
Umd bundle

### PR DESCRIPTION
**Pull request by andreapiras**
## 🖼 Context

This PR adds UMD bundle which is required for legacy system and for SSR rendered env (even though CJS would probably be sufficient for SSR)

## 🚀 Changes

 
Add Rollup
Add babel runtime transform https://github.com/rollup/plugins/tree/master/packages/babel

## ✅ Checklist

- [ ] Make PR title meaningful and follow [the commit lint format](https://github.com/transferwise/neptune-web/blob/master/CONTRIBUTING.md#versioning-and-commit-lint) (it will appear in the changelog)
- [ ] Changes are tested and all tests pass
- [ ] Changes meet [accessibility standards](https://github.com/transferwise/marketing-components/blob/main/ACCESSIBILITY.md) and there are no violations in the console
- [ ] Changes work in all supported browsers (don't forget IE11)
- [ ] You've updated the documentation if necessary
- [ ] Change has been approved by someone outside of your team

### CODE REVIEW

File: packages/marketing-components/src/videomodal/VideoModal.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import * as React from 'react';
 import Types from 'prop-types';
 import { DialogOverlay, DialogContent } from '@reach/dialog';
-import { useTransition, animated } from 'react-spring';
+import { useTransition, animated } from 'react-spring/web.cjs';
```
Comment (andreapiras) [inline packages/marketing-components/src/videomodal/VideoModal.js:4]: react-spring exports by default es which doesn't play well with react
`https://github.com/pmndrs/react-spring/issues/601`

**Review by KarlEST — APPROVED** [review]

File: packages/marketing-components/rollup.config.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+// Rollup
+const input = 'src/index.js';
+const file = pkg.main;
+
+// Rollup can resolve only explicit exports.
+// https://github.com/rollup/rollup/issues/2671
+// https://github.com/rollup/rollup-plugin-commonjs
+const namedExports = {
+  '../../node_modules/@transferwise/formatting/dist/formatting.js': [
```
Comment (twrdjpalmer) [inline packages/marketing-components/rollup.config.js:17]: Where do we use formatting?

Comment (KarlEST) [reply packages/marketing-components/rollup.config.js:17]: I guess this is just copy paste from neptune-web?

Comment (andreapiras) [reply packages/marketing-components/rollup.config.js:17]: yes sorry guys if you'r not using it please remove it

**Pull request merged**